spec/tasks/gias_update_spec.rb

```diff
@@ -5,7 +5,8 @@
   Rails.application.load_tasks if Rake::Task.tasks.empty?
   subject(:gias_update) { Rake::Task["gias_update"].invoke }
 
-  it "calls GiasCsvImporter service" do
+  it "calls Gias::CsvDownloader and Gias::Importer service" do
+    # Testing Gias::CsvDownloader
```

Comment:
In which case, you could use:

```suggestion
  it "runs Gias::SyncAllSchoolsJob" do
    expect { gias_update }.to have_performed_job(Gias::SyncAllSchoolsJob)
```